tests/test_mockgun.py

```diff
@@ -269,6 +272,91 @@ def test_find_by_sub_entity_field(self):
         )
         self.assertEqual(len(items), 1)
 
+    def test_find_by_sub_entity_field_nested(self):
+        """
+        Ensure that queries on nested linked entity fields work.
+        """
+        items = self._mockgun.find("PipelineConfiguration", [
+            ["project.Project.users.HumanUser.login", "is", "user1"]
+        ])
+        self.assertEqual(len(items), 1)
+
+
+class TestFindFields(TestBaseWithExceptionTests):
+    """
+    Ensure that using the 'field' argument in the find method work.
+    """
+
+    def setUp(self):
+        """Creates test data."""
+        self._mockgun = Mockgun("https://test.shotgunstudio.com", login="user", password="1234")
+
+        # Create two users to assign to the pipeline configurations.
+        self._user1 = self._mockgun.create("HumanUser", {"login": "user1", "firstname": "firstname1"})
+
+        # Create a project for nested sub entity field
+        self._project1 = self._mockgun.create("Project", {"name": "project1", "users": [self._user1]})
+
+        # Create pipeline configurations that are assigned none, one or two users.
```

Comment:
Thanks for adding this test, altough looking at the code in master it appears the original code worked. :)